pkg/cli/rook_wait_for_version.go

```diff
@@ -37,7 +37,7 @@ func NewRookWaitForRookVersionCmd(cli CLI) *cobra.Command {
 				return fmt.Errorf("failed to wait for Rook %q: %w", rookVersion, err)
 			}
 
-			fmt.Fprintf(cmd.OutOrStdout(), "\nRook %q has been rolled out\n", rookVersion)
+			fmt.Fprintf(cmd.OutOrStdout(), "Rook %q has been rolled out\n", rookVersion)
```

Comment:
Are you sure this still displays properly?
IIRC the previous line didn't include a newline because it was overwritten each tick (instead of creating a wall of 'not healthy because of X' logs)

Reply:
It had an extra newline for me